src/css/jellyfin.css

```diff
@@ -208,11 +208,11 @@ body {
 }
 
 .fresh {
-    background-image: url(../img/fresh.svg);
+    background-image: url('../img/fresh.svg');
```

Comment:
husky hook wanted single quotes so that's the only change to the css that's left. Now the path gets rewritten instead so that's nice.